modules/database/src/ioc/db/dbCommonPvt.h

```diff
@@ -15,13 +15,22 @@ typedef struct dbCommonPvt {
     /* Thread which is currently processing this record */
     struct epicsThreadOSD* procThread;
 
-    struct dbCommon common;
+    /* actually followed by:
+     * struct dbCommon common;
+     */
 } dbCommonPvt;
+STATIC_ASSERT(sizeof(dbCommonPvt)%8==0);
```

Comment:
Core: `sizeof(dbCommonPvt) % sizeof(void *) == 0`

Reply:
I have replaced this check with:

```c
typedef struct {
    dbCommonPvt prefix;
    /* ensure no padding needed... */
    dbCommon suffix;
} dbCommonPvtAlignmentTest;
STATIC_ASSERT(offsetof(dbCommonPvtAlignmentTest, suffix)==sizeof(dbCommonPvt));
```

As a note, I considered using [gcc/clang `__alignof__`](https://gcc.gnu.org/onlinedocs/gcc-4.8.5/gcc/Alignment.html) and [msvc `alignof`](https://learn.microsoft.com/en-us/cpp/cpp/alignof-operator?view=msvc-170), but decided to avoid non-standard operators in favor of a more verbose test.